src/core/editform/qgsattributeeditorcontainer.h

```diff
@@ -127,6 +146,28 @@ class CORE_EXPORT QgsAttributeEditorContainer : public QgsAttributeEditorElement
      */
     void setVisibilityExpression( const QgsOptionalExpression &visibilityExpression );
 
+    /**
+     * The collapsed expression is used in the attribute form to
+     * set the collapsed status of the group box container container based on an expression incorporating
+     * the field value controlled by editor widgets. This property is ignored if the container is not
+     * rendered as a group box.
+     *
+     * \see setCollapsedExpression()
+     * \since QGIS 3.26
+     */
+    QgsOptionalExpression collapsedExpression() const;
+
+    /**
+     * The collapsed expression is used in the attribute form to
+     * set the collapsed status of the group box of this container based on an expression incorporating
+     * the field value controlled by editor widgets. This property is ignored if the container is not
+     * rendered as a group box.
```

Comment:
```suggestion
     * a group box.
```

(rendered sounds odd here!)